### ISSUE TITLE
genesis new failed with: cf-0.1.8/hooks/params_helper: No such file or directory

### ISSUE THREAD

**Issue by drnic** [genesis-community/cf-genesis-kit#2]
```
/tmp/oYMjm1_LiX/cf-0.1.8/hooks/params: line 3: /tmp/oYMjm1_LiX/cf-0.1.8/hooks/params_helper: No such file or directory
/tmp/oYMjm1_LiX/cf-0.1.8/hooks/params: line 6: $infile: ambiguous redirect
/tmp/oYMjm1_LiX/cf-0.1.8/hooks/params: line 11: contains: command not found
/tmp/oYMjm1_LiX/cf-0.1.8/hooks/params: line 36: $outfile: ambiguous redirect
Error running params hook for cf/0.1.8. Contact your kit author for a bugfix.
```

I see a `hooks/params_helper` on master branch [1] and in v0.1.8 [2]

* [1] https://github.com/genesis-community/cf-genesis-kit/tree/master/hooks
* [2] https://github.com/genesis-community/cf-genesis-kit/tree/0.1.8/hooks

And I see the file in the kit tgz:

```
$ tar tfz .genesis/kits/cf-0.1.8.tar.gz
...
cf-0.1.8/hooks/params
cf-0.1.8/hooks/params_helper
cf-0.1.8/hooks/subkit
```

Bonus question: where is `contains` executable supposed to come from? On my ubuntu machine my `bash` is:

```
$ bash --version
GNU bash, version 4.3.11(1)-release (x86_64-pc-linux-gnu)
```

Perhaps the issue is that `params_helper` isn't executable; but not sure it should be an issue for `source "$(dirname "$0")/params_helper"` 

```
ll hooks
total 32
-rwxr-xr-x  1 drnic  staff   2.0K  1 Aug 10:45 params
-rw-r--r--  1 drnic  staff   7.6K  1 Aug 10:45 params_helper
-rwxr-xr-x  1 drnic  staff   451B  1 Aug 10:45 subkit
```

**Comment by bodymindarts** [comment]
Just got the same issue here!

**Comment by xiujiao** [comment]
high five, same here !

**Comment by geofffranks** [comment]
I'm not seeing this on my laptop, but hte root of the errors is that the `params_helper` script isn't being sourced (it provides all the things that were causing you failures wrt contains + infile/outfile.

I've tried v2.0.7 + v2.1.2 of genesis with success each time. Bash version 3.2.57(1)-release, Perl version v5.18.2.

**Comment by geofffranks** [comment]
Fixed in genesis v2.1.3
